src/SUMMARY.md

```diff
@@ -16,24 +15,26 @@
         - [Running tests](./tests/running.md)
         - [Adding new tests](./tests/adding.md)
         - [Using `compiletest` + commands to control test execution](./compiletest.md)
-    - [Walkthrough: a typical contribution](./walkthrough.md)
-    - [Bug Fix Procedure](./bug-fix-procedure.md)
-    - [Implementing new features](./implementing_new_features.md)
-    - [Stability attributes](./stability.md)
-    - [Stabilizing Features](./stabilization_guide.md)
     - [Debugging the Compiler](./compiler-debugging.md)
     - [Profiling the compiler](./profiling.md)
         - [with the linux perf tool](./profiling/with_perf.md)
-    - [Coding conventions](./conventions.md)
     - [crates.io Dependencies](./crates-io.md)
     - [Errors and Lints](diagnostics.md)
         - [`LintStore`](./diagnostics/lintstore.md)
         - [Diagnostic Codes](./diagnostics/diagnostic-codes.md)
+- [Part 1.5: Contributing to Rust](./contributing.md)
```

Comment:
I think this shows that incorporating "part numbers" into the URL was probably a mistake... not sure of the best way to fix. Maybe just let the part numbers diverge from the URLs? Maybe bite the bullet and break? Anyway, I think that this naming scheme is good as a start.

Reply:
> Maybe bite the bullet and break?

I'm tempted to just do this (but perhaps in another PR). I don't think people tend to link to the part intro chapters, though I might be wrong.

Also, mdbook 0.4 will have support for [breaking the book into parts](https://github.com/rust-lang/mdBook/pull/1171) and doing automatic numbering, so this will not be as much of a problem...